lib/Runtime/ByteCode/Symbol.cpp

```diff
@@ -88,7 +88,7 @@ bool Symbol::IsInSlot(FuncInfo *funcInfo, bool ensureSlotAlloc)
 
 bool Symbol::GetIsCommittedToSlot() const
 {
-    if (!PHASE_ON1(Js::DelayCapturePhase))
+    if (!PHASE_FORCE1(Js::DelayCapturePhase))
```

Comment:
It almost seems like we're missing an optimization opportunity here. Have someone with frontend knowledge  evaluate.